frontend/src/components/common/form/StepForm/SteppedForm.tsx

```diff
@@ -0,0 +1,32 @@
+import { Form, Formik, FormikConfig } from 'formik';
+import * as React from 'react';
+import { StepperFormProvider } from './context';
+import { StepperField } from './StepperField';
+import { ISteppedFormProps, ISteppedFormValues } from './types';
+
+/**
+ * A formik form with a stepper. Use the ```useFormStepper``` hook to access and control the stepper in the form children
+
+ * @component
+ * @example ./SteppedForm.md
+ */
+export const SteppedForm = function<T extends object = {}>({
+  steps,
+  children,
+  initialValues,
+  onSubmit,
+  ...rest
+}: ISteppedFormProps & FormikConfig<ISteppedFormValues<T>>) {
+  return (
+    <Formik<ISteppedFormValues<T>> initialValues={initialValues} onSubmit={onSubmit} {...rest}>
```

Comment:
I'm wondering if having the form as a child is ideal or generic enough.  Wouldn't we also want to support external widgets to be updated or changed based on the stepper stage?  Perhaps child forms should only be one option we support.  Not sure how well React supports listeners though?

Reply:
the hook corresponding to the stepper would handle providing the updates to the child. So the child would call useStepperWhatever and that provides the next and prev actions, current step and whatever other metadata it needs.

Reply:
The children of the form have access to the current state of the stepper by injecting `const stepper = useFormStepper()` and the children can decide on what to display and have buttons to change the stepper state